recipe/meta.yaml

```diff
@@ -20,49 +22,48 @@ requirements:
     - python >=3.6
     - pip
   run:
-  - python >=3.8
+    - python >=3.8
   # Required
-  - affine >=2.2
-  - click >=7.0
-  - cmocean >=2.0
-  - dask >=2021.6
-  - dask-image >=0.6
-  - distributed >=2021.6
-  - fiona >=1.8
-  - fsspec >=2021.6
-  - gdal >=3.0
-  - geopandas >=0.8
-  - jdcal >=1.4.1
-  - jsonschema >=3.2.0
-  - matplotlib-base >=3.0
-  - netcdf4 >=1.5.0
-  - numba >=0.52
-  - numpy >=1.16
-  - pandas >=1.2,<1.3.0  # tests fail with 1.3.0, see xarray #5581
-  - pillow >=6.0
-  - pyjwt >=1.7
-  - pyproj >=3.0
-  - pyyaml >=5.4
-  - rasterio >=1.2
-  - requests >=2.25
-  - requests-oauthlib >=1.3
-  - s3fs >=2021.6
-  - scipy >=1.6.0
-  - setuptools >=41.0
-  - shapely >=1.6
-  - strict-rfc3339 >=0.7  # for python-jsonschema date-time format validation
-  - tornado >=6.0
-  - urllib3 >=1.26
-  # TODO: a lot of tests fail with 0.19
-  # Therefore pin to 0.18.2
-  - xarray =0.18.2
-  - zarr >=2.8
+    - affine >=2.2
+    - click >=8.0
+    - cmocean >=2.0
+    - dask >=2021.6
+    - dask-image >=0.6
+    - distributed >=2021.6
+    - fiona >=1.8
+    - fsspec >=2021.6
+    - gdal >=3.0
+    - geopandas >=0.8
+    - jdcal >=1.4.1
+    - jsonschema >=3.2.0
+    - matplotlib-base >=3.0
+    - netcdf4 >=1.5.0
+    - numba >=0.52
+    - numpy >=1.16
+    - pandas >=1.3
+    - pillow >=6.0
+    - pip
+    - pyjwt >=1.7
+    - pyproj >=3.0
+    - pyyaml >=5.4
+    - rasterio >=1.2
+    - requests >=2.25
+    - requests-oauthlib >=1.3
+    - s3fs >=2021.6
+    - scipy >=1.6.0
+    - setuptools >=41.0
+    - shapely >=1.6
+    - strict-rfc3339 >=0.7 # for python-jsonschema date-time format validation
+    - tornado >=6.0
+    - urllib3 >=1.26
+    - xarray >=0.19
+    - zarr >=2.8
   # Testing
-  - flake8 >=3.7
-  - moto >=1.3
-  - pytest >=4.4
-  - pytest-cov >=2.6
-  - requests-mock >=1.8.0
+    - flake8 >=3.7
+    - moto >=1.3
+    - pytest >=4.4
+    - pytest-cov >=2.6
+    - requests-mock >=1.8.0
```

Comment:
Hi! Just trying xcube for the first time. I'm wondering if these can be removed from meta.yaml as they are only needed for development. As a *user* of xcube I'd like to be able to `conda install xcube` and not bring in libraries required only for testing and development.

Reply:
Thank you, good catch! You're right, these shouldn't be here, and I've now removed them.